src/components/organisms/notifier/NotificationChannelsList.tsx

```diff
@@ -23,17 +21,16 @@ const useStyles = makeStyles((theme: Theme) => createStyles({
     borderTop: `1px solid ${colors.gray3}`,
     padding: theme.spacing(3, 0),
   },
-
 }))
 
 type Props = {
     channels?: Array<SupportedEventChannel>
+    setEventChannels: (channels: Array<NotifierChannel>) => void
 }
 
-const NotificationChannelsList: FC<Props> = ({ channels }) => {
-  const [addedChannels, setChannels] = useState<Array<NotifierChannel>>([])
-
+const NotificationChannelsList: FC<Props> = ({ channels, setEventChannels }) => {
   const classes = useStyles()
+  const [addedChannels, setChannels] = useState<Array<NotifierChannel>>([])
```

Comment:
Please, use same name for updater and callback for easy reading.

```suggestion
  const [addedChannels, setAddedChannels] = useState<Array<NotifierChannel>>([])
```

Reply:
done